crates/modelardb_server/src/storage/mod.rs

```diff
@@ -106,7 +107,7 @@ impl StorageEngine {
         local_data_folder: PathBuf,
         remote_data_folder: Option<Arc<dyn ObjectStore>>,
         configuration_manager: &Arc<RwLock<ConfigurationManager>>,
-        metadata_manager: Arc<MetadataManager>,
+        metadata_manager: Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.